CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## To be Released
+## 1.0.4
```

Comment:
The version bump is not supposed to happen in this PR. It will be part of the release of a new version (https://github.com/Scalingo/gopassword?tab=readme-ov-file#release-a-new-version).

Reply:
Oups, revert is coming...